Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM openjdk:16-alpine
 RUN apk add --no-cache bash tar && \
-         adduser consignment-export -D
+    adduser consignment-export -D
 WORKDIR /home/consignment-export
 USER consignment-export
-COPY target/universal/tdr-consignment-export.tgz ./tdr-consignment-export.tgz
+COPY exporter/target/universal/tdr-consignment-export.tgz ./tdr-consignment-export.tgz
```

Comment:
Should the authoriser module also be copied to the image?

Reply:
No, the authoriser module is deployed as a lambda, this docker image is for the ECS task that does the export.